fluent-package/convert-artifacts-layout.sh

```diff
@@ -15,17 +15,17 @@ case $1 in
     apt|deb)
 	REPOSITORY_TYPE=apt
 	REPOSITORY_PATH=$TD_AGENT_DIR/$REPOSITORY_TYPE/repositories
-	for d in buster bullseye bionic focal jammy; do
+	for d in bullseye bookworm focal jammy; do
 	    case $d in
-		buster|bullseye)
+		bullseye)
```

Comment:
Although `bookworm` is added to the list, there is no case for `bookworm`.
Is this expected?
Is it because we don't support it yet?

Reply:
I've overlooked it, thanks.